packages/select/src/createDeletePlugin.ts

```diff
@@ -0,0 +1,17 @@
+import { createPluginFactory, QueryNodeOptions } from '@udecode/plate-common';
+
+import { withDelete } from './withDelete';
+
+export type DeletePlugin = {
+  query?: QueryNodeOptions;
+};
+
+export const KEY_DELETE = 'delete';
+
+/**
+ * @see {@link withDelete}
+ */
+export const createDeletePlugin = createPluginFactory<DeletePlugin>({
+  key: KEY_DELETE,
+  withOverrides: withDelete,
```

Comment:
The default query should be here. There should be no options in `plate-plugins.ts`.

Reply:
and default value will be ELEMENT_DEFAULT?


ps: thank you for bearing with my silly mistakes, Its my first open source contribution.

Reply:
It should be the same `options` object you had in the plate-plugins.ts, but with ELEMENT_DEFAULT, yes.

> ps: thank you for bearing with my silly mistakes, Its my first open source contribution.

It's no trouble ❤️